src/games/mod.rs

```diff
@@ -634,6 +645,36 @@ pub fn build_character_id_map() -> anyhow::Result<HashMap<u32, String>> {
     Ok(result)
 }
 
+#[derive(serde::Deserialize)]
+#[allow(non_snake_case)]
+pub struct AvatarSkillDepotExcelConfigDataEntry {
+    pub energySkill: u32,
+    pub skills: Vec<u32>,
+}
+
+#[derive(Copy, Clone, Debug)]
+enum SkillType {
```

Comment:
I got the following warning :
```
warning: type `SkillType` is more private than the item `build_skill_type_map`
```

I think you just need to add a `pub`.
```suggestion
pub enum SkillType {
```

Reply:
done